draft-ietf-ace-cwt-proof-of-possession.xml

```diff
@@ -274,21 +274,21 @@
 	  the <spanx style="verb">COSE_Key</spanx> member
 	  is a COSE_Key <xref target="RFC8152"/>
 	  representing the corresponding asymmetric public key.
-	  The following example (using JSON notation) demonstrates such a declaration
+	  The following example (using CBOR diagonstic notation) demonstrates such a declaration
 	  in the CWT Claims Set of a CWT:
 	</t>
 	<figure>
 	  <artwork><![CDATA[
-  {
-   "iss": "https://server.example.com",
-   "aud": "https://client.example.org",
-   "exp": 1361398824,
-   "cnf":{
-     "COSE_Key":{
-       "kty": "EC",
-       "crv": "P-256",
-       "x": "18wHLeIgW9wVN6VD1Txgpqy2LszYkMf6J8njVAibvhM",
-       "y": "-V4dS4UaLMgP_4fY4j8ir7cl1TXlFdAgcx55o7TkcSA"
+   {
+   /iss/ 1 : "coaps://server.example.com",
+   /aud/ 3 : "coaps://client.example.org",
+   /exp/ 4 : 1361398824,
+   /cnf/ 8 :{
+     /COSE_Key/ 1 :{
+       /kty/ 1 : /EC/ 2,
+       /crv/ -1 : /P-256/ 1,
+       /x/ -2 : b64'EtboxNKPgxEKV9JTNzytUvAbxEfkCTVB9kOzheF5wRA',
+       /y/ -3: b64'KDs9jSj_pZ_ly1QEEqdQ-o36NPbaabzaaEANZ5wTR-g'
```

Comment:
It doesn't really matter, but is there a specific reason you changed the x and y values?